cmd/allocator/metrics.go

```diff
@@ -34,19 +35,23 @@ const (
 	stackdriverLabels            = "stackdriver-labels"
 	mTLSDisabledFlag             = "disable-mtls"
 	tlsDisabledFlag              = "disable-tls"
+	allocationTimeoutFlag        = "allocation-timeout"
+	totalAllocationTimeoutFlag   = "total-allocation-timeout"
 )
 
 func init() {
 	registerMetricViews()
 }
 
 type config struct {
-	TLSDisabled       bool
-	MTLSDisabled      bool
-	PrometheusMetrics bool
-	Stackdriver       bool
-	GCPProjectID      string
-	StackdriverLabels string
+	TLSDisabled            bool
+	MTLSDisabled           bool
+	PrometheusMetrics      bool
+	Stackdriver            bool
+	GCPProjectID           string
+	StackdriverLabels      string
+	TotalAllocationTimeout time.Duration
+	AllocationTimeout      time.Duration
```

Comment:
The naming is not clear when it is set for the agones-allocator service. It seems that it applies to a single cluster allocation. How about renaming it to *RemoteAllocationTimeout?

Reply:
Will change